templates/base.html.twig

```diff
@@ -5,11 +5,10 @@
         <title>{% block title %}Welcome!{% endblock %}</title>
         <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 128 128%22><text y=%221.2em%22 font-size=%2296%22>⚫️</text></svg>">
         {% block stylesheets %}
-            <link rel="stylesheet" href="{{ asset('styles/app.css') }}">
         {% endblock %}
 
         {% block javascripts %}
-            {{ importmap() }}
+{% block importmap %}{{ importmap('app') }}{% endblock %}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏We need to run `symfony console importmap:install` for this to work now, could you add it to the readme?

Reply:
It'd be better in composer.json scripts, yes?

I'm away from my computer, back on Wednesday, I'll do both of these then.

Reply:
I think so to!

And sure no problem, there's really no rush ^^

Reply:
Could you fix the indentation?